policy/executor/internal/builder.go

```diff
@@ -192,14 +194,21 @@ func (b *GraphBuilder) Build(runtime llx.Runtime, assetMrn string) (*GraphExecut
 		}
 	}
 
+	reportingQueryForReportingJob := map[string]string{}
 	for queryID, q := range queries {
 		canRun := checkVersion(q.codeBundle, mondooVersion)
 		if canRun {
 			ge.addExecutionQueryNode(queryID, q, q.resolvedProperties, b.datapointType)
 		} else {
 			unrunnableQueries = append(unrunnableQueries, q)
 		}
-		ge.addReportingQueryNode(queryID, q)
+		n := ge.addReportingQueryNode(queryID, q)
+		if len(q.notifies) > 0 {
+			for _, notify := range q.notifies {
+				ge.addEdge(n.id, NodeID(notify))
```

Comment:
creates the edge from the node we use to represent a reporting query to the node we use to represent the reporting job. This will be the reporting job for query code id

Reply:
Also note that the reporting query node doesn't fully match whats in the resolved policy. We create one of these for every query, scoring or data. Only scoring queries are in the reporting queries section of the resolved policy.